java/java.source.base/src/org/netbeans/modules/java/source/usages/BinaryAnalyser.java

```diff
@@ -284,9 +285,10 @@ public IdentLevel getIdentLevel() {
                 return idLvl;
             }
 
+            @NbBundle.Messages("USE_FULL_INDEX=false")
```

Comment:
Usually we document this API in `arch.xml` like in case of #2848.

Reply:
`USE_FULL_INDEX` branding token documented as requested.